client/src/components/AddProjectModal.jsx

```diff
@@ -52,6 +52,7 @@ export default function AddProjectModal() {
             className='btn btn-primary'
             data-bs-toggle='modal'
             data-bs-target='#addProjectModal'
+            disabled={data.clients.length < 1 ? 'disabled' : ''}
```

Comment:
Just seen this by chance, but since `disabled` expects a boolean, the following would probably make more sense:

```suggestion
            disabled={data.clients.length === 0}
```